docs/index.md

```diff
@@ -1,6 +1,14 @@
 # CCHDO Documentation
 
-Welcome, explore our resources and guides below: 
+Welcome to the CCHDO documentation site!
+
+This is the learning companion site to the CLIVAR and Carbon Hydrographic Data Office (CCHDO) found at [cchdo.ucsd.edu](https://cchdo.ucsd.edu). For data downloads and cruise information, please visit [cchdo.ucsd.edu](https://cchdo.ucsd.edu).
+
+Information about file formats, our parameters, and data usage notebooks can be found under **"Getting Started"**. Our submission guidelines with format templates and commonly asked questions are available under **"Submitting Data to CCHDO"**. Citation information, DOI, and licenses can be found under **"Policies and Procedures"**, along with additional information and guides.
```

Comment:
I think these should be links to the content: for example `**"Getting Started"**` should turn into `**"<project:./getting_started/index.md>"**`.

If there is an explicit reason for not wanting this, just ignore/resolve this comment.

Reply:
This can also be done with a `**"[](./getting_started/index.md)"**` syntax I think.

Reply:
Yeah I think I agree with this

Reply:
Added in [3019416](https://github.com/cchdo/cchdo-learning-and-guides/pull/12/commits/30194164d5a0f6f28b0c8c3878be7d8df5ac2dd6)